### ISSUE TITLE
Support public_ip

### ISSUE THREAD

**Issue by R1KO** [yourmnbbn/tiny-csgo-server#19]
Add argument for specify server public ip. Original srcds  args: `-ip`, `+net_public_addr`

Because when i'm running server in docker container server starts on local container ip and can't receive connects.

**Comment by saguaro-yerba** [comment]
You can achieve this by creating iptables rules

**Comment by R1KO** [comment]
Can you elaborate? I do not understand how iptables will help with this

**Comment by yourmnbbn** [comment]
Haven't tested in docker, I'll take a look at this when I have time.

**Comment by saguaro-yerba** [comment]
> Can you elaborate? I do not understand how iptables will help with this

If the container has an internet connection and you just need to forward the udp port.
I also run this on Docker on my local network and all I need is UDP port forwarding:
`docker run -p 27015:27015/udp [...]`
Your server should be visible from Steam Masterserver API:
http://api.steampowered.com/ISteamApps/GetServersAtAddress/v0001?addr=<SERVER PUBLIC IP HERE>&format=json

**Comment by R1KO** [comment]
> > Can you elaborate? I do not understand how iptables will help with this
> 
> If the container has an internet connection and you just need to forward the udp port. I also run this on Docker on my local network and all I need is UDP port forwarding: `docker run -p 27015:27015/udp [...]` Your server should be visible from Steam Masterserver API: http://api.steampowered.com/ISteamApps/GetServersAtAddress/v0001?addr=&format=json

It works! But it require specify different ports for each container because add to master server  with port specified in `port` option =(

**Comment by saguaro-yerba** [comment]
> > > Can you elaborate? I do not understand how iptables will help with this
> > 
> > 
> > If the container has an internet connection and you just need to forward the udp port. I also run this on Docker on my local network and all I need is UDP port forwarding: `docker run -p 27015:27015/udp [...]` Your server should be visible from Steam Masterserver API: http://api.steampowered.com/ISteamApps/GetServersAtAddress/v0001?addr=&format=json
> 
> It works! But it require specify different ports for each container because add to master server with port specified in `port` option =(

I'm not sure if I understand correctly, but you can try using host network `--network host`, then you don't need to do redirect UDP ports on containers

**Comment by R1KO** [comment]
> I'm not sure if I understand correctly, but you can try using host network `--network host`, then you don't need to do redirect UDP ports on containers

i know) I have done so now, but I'm afraid of possible problems associated with it.

Thanks for response)

**Comment by saguaro-yerba** [comment]
> > I'm not sure if I understand correctly, but you can try using host network `--network host`, then you don't need to do redirect UDP ports on containers
> 
> i know) I have done so now, but I'm afraid of possible problems associated with it.
> 
> Thanks for response)

Yeah that's true, it can cause some problems, port forwarding on Docker worked better for me 😀
